### PR TITLE
fix(xss): improve sanitize and add test cases

### PR DESCRIPTION
**Pull request by robertu7**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -95,11 +96,12 @@
     "request-ip": "^2.1.3",
     "stripe": "^8.144.0",
     "typescript": "^3.8.3",
+    "utf-8-validate": "^5.0.2",
```
Comment (robertu7) [inline package.json:99]: added by `npm audit fix`

File: package.json [hunk 2]
```diff
@@ -53,6 +53,7 @@
     "axios": "^0.21.1",
     "bcrypt": "^5.0.0",
     "bodybuilder": "^2.2.20",
+    "bufferutil": "^4.0.1",
```
Comment (robertu7) [inline package.json:56]: added by `npm audit fix`

**Review by guoliu — APPROVED** [review]
Nice 🍀

**Review by devformatters — APPROVED** [review]
Go. 🌂

**Pull request merged**